BraveRewardsUI/Settings/Tips/Details/TipsDetailViewController.swift

```diff
@@ -36,6 +39,32 @@ class TipsDetailViewController: UIViewController {
     tipsView.tableView.dataSource = self
 
     title = Strings.Tips
+    loadData()
+  }
+  
+  private func loadData() {
+    _ = getTipsThisMonth().then {
+      totalBatTips = $0.total.isEmpty ? "0.0" : $0.total
```

Comment:
You aren't checking for tips here? Total can be zero if they were to tip 10 BAT and then receive 10 BAT from ads/grants.  You should be creating `BATValue` objects from `oneTimeDonation` and `recurringDonation` and checking those

Reply:
Oh I thought total was that. Thanks for the info